app/views/stripe-setup/responsible-person/check-your-answers.njk

```diff
@@ -0,0 +1,111 @@
+{% extends "../../layout.njk" %}
+{% from "components/summary-list/macro.njk" import govukSummaryList %}
+{% from "components/button/macro.njk" import govukButton %}
+
+{% block pageTitle %}
+  Nominate a responsible person - {{ currentService.name }} - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-two-thirds-from-desktop govuk-!-margin-top-6">
+
+    <span id="request-to-go-live-current-step" class="govuk-caption-l">Before you can take payments</span>
+    <h1 class="govuk-heading-l">Check details before saving</h1>
+
+    <form action="/responsible-person" method="POST" novalidate>
+      <input id="csrf" name="csrfToken" type="hidden" value="{{ csrf }}">
+      <input name="answers-need-changing" type="hidden" value="true">
+      <input name="first-name" type="hidden" value="{{ firstName }}">
+      <input name="last-name" type="hidden" value="{{ lastName }}">
+      <input name="home-address-line-1" type="hidden" value="{{ homeAddressLine1 }}">
+      <input name="home-address-line-2" type="hidden" value="{{ homeAddressLine2 }}">
+      <input name="home-address-city" type="hidden" value="{{ homeAddressCity }}">
+      <input name="home-address-postcode" type="hidden" value="{{ homeAddressPostcode }}">
+      <input name="dob-day" type="hidden" value="{{ dobDay }}">
+      <input name="dob-month" type="hidden" value="{{ dobMonth }}">
+      <input name="dob-year" type="hidden" value="{{ dobYear }}">
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            First name
+          </dt>
+          <dd class="govuk-summary-list__value">
+            {{ firstName }}
+          </dd>
+          <dd class="govuk-summary-list__actions">
+            <button class="govuk-button--as-link" type="submit">
+              Change
+              <span class="govuk-visually-hidden">first name</span>
+            </button>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Last name
+          </dt>
+          <dd class="govuk-summary-list__value">
+            {{ lastName }}
+          </dd>
+          <dd class="govuk-summary-list__actions">
+            <button class="govuk-button--as-link" type="submit">
+              Change
+              <span class="govuk-visually-hidden">last name</span>
+            </button>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Home address
+          </dt>
+          <dd class="govuk-summary-list__value">
+            {{ homeAddressLine1 }}
+            <br>
+            {{ homeAddressLine2 }}
+            <br>
+            {{ hoomeAddressCity }}
```

Comment:
Hoooooome

Reply:
lool good spot, not sure how we missed that. Thanks